tools/db_bench_tool.cc

```diff
@@ -7340,6 +7351,42 @@ int db_bench_tool(int argc, char** argv) {
     FLAGS_env = new ROCKSDB_NAMESPACE::HdfsEnv(FLAGS_hdfs);
   }
 
+#ifndef ROCKSDB_LITE
+  // Check for the encryption flag (and create an encrypted environment)
+  Random rnd(301);
+  std::shared_ptr<EncryptionProvider> provider;
+  if (FLAGS_encrypt_db && !strcasecmp(FLAGS_block_cipher.c_str(), "IPP_AES")) {
```

Comment:
If you need the IPP_AES here, can you make it a constant somewhere that is accessible by multiple places?  Perhaps EncryptionProvider::kIPPAESName()?

Reply:
Sure.